examples/quiz-robust/qwiklabs.yml

```diff
@@ -0,0 +1,105 @@
+entity_type: Quiz
+schema_version: 1
+
+title: Robust Quiz
+passing_percentage: 67
+
+items:
+  - type: multiple-choice
+    stem:
+      locales:
+        en: Who won the Nobel Prize for discovering the cause of the photoelectric effect?
+        fr: Qui a remporté le prix Nobel pour avoir découvert la cause de l'effet photoélectrique?
+    options:
+      - title:
+          locales:
+            en: Albert Einstein
+            fr: Albert Einstein
+        is_answer: true
```

Comment:
Prefixing these with `is_` feels out of place in the spec. I'd lean towards just `answer: true` and `answer: false`.

Depending on where quizzes are going and how complicated questions are going to get, you may want to change this to something like `answer_type: {answer,distractor,hard_distractor}`

Reply:
I'm not wild about `answer: true`, since that's also how we define the boolean answer to a true false item. I'm not sure I follow on `answer_type`, but I can say that if the answers got more complex it almost certainly would be an entirely new item type (which is likely to happen and is a good thing!) and not adding complexity to MC, just like how MS is not a property on MC but an entirely separate type.

Reply:
This may require a bit of mental gymnastics, however...

This actually matches the `true-false` use of `answer` pretty well. For each of these individual options, they each have a correct `true`/`false` value.

Reply:
@jgan yep, that's definitely true. I don't think we should go down that route, though - more explicit is better when it comes to these item types.